### PR TITLE
pulumi-version: add page

### PR DESCRIPTION
**Pull request by nelsonfigueroa**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

### CODE REVIEW

File: pages/common/pulumi-version.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+> Print Pulumi’s version number.
+> More information: <https://www.pulumi.com/docs/iac/cli/commands/pulumi_version/>.
+
+- Print Pulumi's version number:
+
+`pulumi version`
+
+- Display help:
+
+`pulumi version {{-h|--help}}`
```
Comment (spageektti) [inline pages/common/pulumi-version.md:12]:
```suggestion
# pulumi version

> Display Pulumi’s version number.
> More information: <https://www.pulumi.com/docs/iac/cli/commands/pulumi_version/>.

- Display version:

`pulumi version`

- Display help:

`pulumi version {{-h|--help}}`
```
we use display here too, like in the help example, see styleguide

**Review by spageektti — COMMENTED** [review]
hi, thanks for your contribution

File: pages/common/pulumi-version.md (hunk 1 above)
Comment (nelsonfigueroa) [reply pages/common/pulumi-version.md:12]: I should know better :)

File: pages/common/pulumi-version.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# pulumi version
+
+> Display Pulumi’s version number.
```
Comment (gutjuri) [inline pages/common/pulumi-version.md:3]:
```suggestion
> Display Pulumi's version number.
```

**Review by gutjuri — APPROVED** [review]
Thanks, however, i'd use the single quote character here

**Review by spageektti — APPROVED** [review]

**Pull request merged**